### DISCUSSION TITLE
MODIS LAI STAC ('modis-15A2H-061') may have broken time metadata?

### DISCUSSION THREAD

**Discussion by erthward** [microsoft/PlanetaryComputer#102] [Q&A] [answered]
Hi, MSPC.

I'm prototyping some work on the Hub, and I noticed that when I used the same basic code to load and peruse DataArrays from different collections I got null time data returned for the MODIS LAI dataset ('modis-15A2H-061'). I don't believe this would be expected given my code, especially given that identical code returns working data for other collections, so I took this as a sign that something is wrong with the STAC, but of course please let me know if I'm doing something dumb on my end!

Sample code to reproduce:
```python
import pystac_client, planetary_computer

catalog = pystac_client.Client.open("https://planetarycomputer.microsoft.com/api/stac/v1")
bbox = [-79.21787, 37.25838, -79.20933, 37.26692] # ~8km ENE of Evington, VA
dt = "2018-01-01/2022-05-09"

search = catalog.search(
        collections=['modis-15A2H-061'],
        bbox=bbox,
        datetime=dt,
    )

items = search.get_all_items()

da = stackstac.stack(planetary_computer.sign(items),
                                 bounds_latlon=bbox,
                                 epsg=4326,
                                )

print(da.time)
```

Screenshot of resulting problem:

![image](https://user-images.githubusercontent.com/6105510/189435175-46e04106-2034-4a7f-8056-8ed703e7ccae.png)

Thanks!

**Comment by TomAugspurger** [comment, marked as answer]
Thanks for the easily reproducible example.

The MODIS items (I think all of the collections) use a `null` datetime since they are composites over some time period. If you check `items[0].datetime` you'll see it's None, while `items[0].properties["start_datetime"]` and end_datetime are valid.

The `time` dimension returned by `stackstac` is comes from the `datetime`. The properties like `start_datetime` should be available as coordinates, and you should be able to make them a dimension with something like `da.swap_dims({"time": "start_datetime"})`

**Reply by erthward** [reply]
Oh, perfect! That's what I've been doing in the meantime, but I thought it was a hack.
Thanks, Tom!

D
